Exercises/data-types-overview/data-types-overview.js

```diff
@@ -0,0 +1,84 @@
+// Declare a variable on each type using two options: plain creation, creation with constructor.
+
+//number
+const number = 123;
+const number2 = Number('456');
+//string
+const string = 'some string';
+const string2 = String('new string');
+
+//boolean
+const booleanVariable = true;
+const booleanVariable2 = Boolean(false);
```

Comment:
When you use `Number/String/Boolean` without `new` keyword before it is not a constructor but just a function. You can read some facts about the difference here:
https://stackoverflow.com/a/4719342
In this task constructors should be used, not functions.